src/test_helpers/tests/integration_test.cc

```diff
@@ -0,0 +1,129 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include <deal.II/base/utilities.h>
+#include <deal.II/base/logstream.h>
+
+#include <sys/stat.h>
+#include <exception>
+#include <fstream>
+
+
+
+#include "../bart_test_helper.h"
+
+using ::testing::MatchesRegex;
```

Comment:
I guess Rachel has talked about namespace several times. If possible, please explicitly specify namespace each time we call something from it instead of `using namespace xxx`. I did feel confused when reading the following code until I noticed this. If the name space is too long, we might just rename it. The purpose is to enhance the readability and understandability of the code for other people.